scripts/cmake/vcpkg_check_features.cmake

```
@@ -89,6 +89,7 @@ function(vcpkg_check_features)
         if(variable STREQUAL last_variable)
             message(FATAL_ERROR "vcpkg_check_features passed the same feature variable multiple times: '${variable}'")
```

```suggestion
            message("${Z_VCPKG_BACKCOMPAT_MESSAGE_LEVEL}" "vcpkg_check_features passed the same feature variable multiple times: '${variable}'")
```

Given that it was toasted before I think we need to let existing callers of old versions continue to work.